pkg/resources/user_grant.go

```diff
@@ -13,7 +13,6 @@ import (
 
 var validUserPrivileges = NewPrivilegeSet(
 	privilegeMonitor,
-	privilegeOwnership,
```

Comment:
I remove `privilegeOwnership` as it is already being serviced by `UserOwnershipGrant` resource.
Leaving it in may contribute to confusion.